doc/install-macos.md

```diff
@@ -0,0 +1,78 @@
+# macOS Installation
+
+To install the fluxon-mhd modeling framework, there are three main requirements - a perl installation with libraries, the perl data language toolset, and finally compiling the fluxon-mhd software itself. Some of the background perl installation requires root / admin access, but beyond that packages can be installed locally for ease of editing and recompilation.
+
+## Perl installation
+
+First, install the homebrew package, which will require sudo access.
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Next, use homebrew as an admin user to install perl.
+```shell
+brew install perl
+```
+
+If you plan on using homebrew for other packages, and would like to keep perl from updating alongside other packages, the installation can be frozen.
+```shell
+brew pin perl
+```
+
+Two more homebrew packages will be needed later, which can also be installed via an admin user.
+```shell
+brew install gnuplot
```

Comment:
I think that brew install can take multiple packages as arguments to reduce the number of commands. I.e. brew install gnuplot fftw. This is a stylistic choice I'll leave to you!